datacenter/ucp/2.0/guides/installation/system-requirements.md

```diff
@@ -31,8 +31,8 @@ When installing UCP on a host, make sure the following ports are open:
 | managers, workers |    in     | TCP 443  (configurable) | Port for the UCP web UI and API                                                   |
 | managers          |    in     | TCP 2376 (configurable) | Port for the Docker Swarm manager. Used for backwards compatibility               |
 | managers, workers |    in     | TCP 2377 (configurable) | Port for communication between swarm nodes                                        |
-| managers, workers |  in, out  | UDP 4789                | Port for overlay networking                                                       |
-| managers, workers |  in, out  | TCP, UDP 7946           | Port  for overlay networking                                                      |
+| managers, workers |  in, out  | TCP, UDP 4789           | Port for overlay networking                                                       |
```

Comment:
According to the [swarm docs](https://docs.docker.com/engine/swarm/swarm-tutorial/#the-ip-address-of-the-manager-machine) port 4789 is just UDP, so I'm thinking that #1683 is probably wrong on that one. Can you remove the mention to TCP?